content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,89 @@
+# Continuous Integration and Deployment for Machine Learning
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers), testing, and deployment to production (CI/CD --
+continuous integration and continuous delivery), as well as monitoring &
+feedback.
+
+![](https://static.iterative.ai/img/ml-pipeline.png) _Basic ML pipeline_
+
+The main benefits of CI/CD for ML are detailed below.
```

Comment:
Should we mention http://CML.dev project in the first paragraph? It will look more practical with the reference.